frontend/src/metabase-lib/queries/StructuredQuery.ts

```diff
@@ -538,6 +538,10 @@ class StructuredQueryInner extends AtomicQuery {
     return this.breakouts().length > 0;
   }
 
+  hasAnyBreakouts() {
+    return this.queries().some(query => query.hasBreakouts());
```

Comment:
Should we also check `this.query()` here?

I mean

```suggestion
    return [this.query(), ...this.queries()].some(query => query.hasBreakouts());
```

or

```suggestion
    return this.hasBreakouts() || this.queries().some(query => query.hasBreakouts());
```

----

If that's the case, then I'd call this function `anyQueryHasBreakouts`.